internal/metarepos/raft_metadata_repository_test.go

```diff
@@ -489,6 +489,64 @@ func TestMRApplyReport(t *testing.T) {
 	})
 }
 
+func TestMRApplyInvalidReport(t *testing.T) {
+	Convey("Given LogStream", t, func(ctx C) {
+		rep := 2
+		clus := newMetadataRepoCluster(1, rep, false, false)
+		Reset(func() {
+			clus.closeNoErrors(t)
+		})
+		mr := clus.nodes[0]
+
+		tn := &varlogpb.TopicDescriptor{
+			TopicID: types.TopicID(1),
+			Status:  varlogpb.TopicStatusRunning,
+		}
+
+		err := mr.storage.registerTopic(tn)
+		So(err, ShouldBeNil)
+
+		snIDs := make([]types.StorageNodeID, rep)
+		for i := range snIDs {
+			snIDs[i] = types.MinStorageNodeID + types.StorageNodeID(i)
+
+			sn := &varlogpb.StorageNodeDescriptor{
+				StorageNode: varlogpb.StorageNode{
+					StorageNodeID: snIDs[i],
+				},
+			}
+
+			err := mr.storage.registerStorageNode(sn)
+			So(err, ShouldBeNil)
+		}
+		err = mr.storage.registerTopic(&varlogpb.TopicDescriptor{TopicID: types.TopicID(1)})
+		So(err, ShouldBeNil)
```

Comment:
There is another line to register the topic above; L506-507. Is this different?

Reply:
deleted